Cargo.toml

```
@@ -12,8 +12,9 @@ categories = ["gui"]
 rust-version = "1.71"
 
 [features]
-default = ["libxdo"]
-libxdo = ["dep:libxdo"]
+default = ["libxdo", "gtk"]
+libxdo = ["dep:libxdo", "gtk"]
```

i think this can stay as is
```suggestion
libxdo = ["dep:libxdo"]
```